pkgs/applications/networking/cluster/minikube/default.nix

```diff
@@ -1,9 +1,31 @@
-{ stdenv, buildGoPackage, fetchFromGitHub, fetchurl, go-bindata, kubernetes, libvirt, qemu, docker-machine-kvm,
+{ stdenv, buildGoPackage, fetchFromGitHub, fetchurl, go-bindata, kubernetes, libvirt, pkgconfig, qemu, docker-machine-kvm,
   gpgme, makeWrapper, hostPlatform, vmnet }:
 
 let
+  version = "0.26.0";
+  src = fetchFromGitHub {
+    owner  = "kubernetes";
+    repo   = "minikube";
+    rev    = "v${version}";
+    sha256 = "1wc2gvmgb59yh0ldm2plvh6s8mvxvysrxp6w75z16ii86jmi3wr6";
+  };
+
+  docker-machine-kvm2 = buildGoPackage rec {
```

Comment:
Could you move out this expression and put it close to the `docker-machine-kvm` attribute?